src/frontend/src/pages/DesignReviewView.tsx

```diff
@@ -0,0 +1,186 @@
+        {Array.from({ length: 6 }, (_, i) => (
+          <Box
+            sx={{
+              width: '1.5vw',
+              height: '1.5vw',
+              backgroundColor: colors[i]
+            }}
+          />
+        ))}
+        <Typography style={{ marginLeft: '10px', fontFamily: 'oswald' }}>
+          {totalUsers}/{totalUsers}
+        </Typography>
+      </Grid>
+    );
+  };
+
+  const renderAvailabilites = () => {
+    const fontSize = totalUsers > 10 ? '1em' : totalUsers > 15 ? '0.8em' : '1.2em';
+    return (
+      <Grid
+        style={{
+          display: 'flex',
+          flexDirection: 'column',
+          backgroundColor: '#2C2C2C',
+          padding: '20px',
+          borderRadius: '8px',
+          height: '100%',
+          overflow: 'auto'
+        }}
+      >
+        <Grid style={{ display: 'flex', flexDirection: 'column', flexGrow: 1, marginBottom: '10px' }}>
+          {renderLegend()}
+          <Grid style={{ display: 'flex', gap: '2', marginTop: '10px' }}>
+            <Grid>
+              <Typography
+                style={{
+                  textDecoration: 'underline',
+                  fontFamily: 'oswald',
+                  fontSize: '1.5em',
+                  textAlign: 'center',
+                  marginBottom: '10px'
+                }}
+              >
+                Available Users
+              </Typography>
+              {currentAvailableUsers.map((user) => (
+                <Typography style={{ fontFamily: 'oswald', textAlign: 'center', fontSize }}>
+                  {user.firstName} {user.lastName}
+                </Typography>
+              ))}
+            </Grid>
+            <Grid>
+              <Typography
+                style={{
+                  textDecoration: 'underline',
+                  fontFamily: 'oswald',
+                  fontSize: '1.5em',
+                  textAlign: 'center',
+                  marginBottom: '10px'
+                }}
+              >
+                Unvailable Users
+              </Typography>
+              {currentUnavailableUsers.map((user) => (
+                <Typography style={{ fontFamily: 'oswald', textAlign: 'center', fontSize }}>
+                  {user.firstName} {user.lastName}
+                </Typography>
+              ))}
+            </Grid>
+          </Grid>
+          <Grid
+            style={{
+              marginTop: 'auto',
+              display: 'flex',
+              justifyContent: 'flex-end',
+              gap: '10px'
+            }}
+          >
+            <WarningIcon style={{ color: 'yellow', fontSize: '2em', marginTop: '5px' }} />
+            <NERFailButton>Cancel</NERFailButton>
+            <NERSuccessButton>Finalize</NERSuccessButton>
+          </Grid>
+        </Grid>
+      </Grid>
+    );
+  };
+
+  return (
+    <Grid display={'flex'}>
+      {renderSchedule()}
+      <Grid>{renderAvailabilites()}</Grid>
+    </Grid>
+  );
+};
+
```

Comment:
this happens when the screen is too small and the availabilities section is larger than the availability section. You may need to just set a maxHeight on the availability view:
<img width="936" alt="image" src="https://github.com/Northeastern-Electric-Racing/FinishLine/assets/29521172/10fce3e1-60c4-4f28-8877-db33fe307451">